mm2src/lp_swap/check_balance.rs

```diff
@@ -255,7 +255,7 @@ impl CheckBalanceError {
                     }
                 } else {
                     CheckBalanceError::NotSufficientBaseCoinBalance {
-                        coin,
+                        coin: ticker.to_string(),
```

Comment:
`TradePreimageError::NotSufficientBalance::coin` is the coin, the balance of which is not enough.
So I think the problem is in the place where the `TradePreimageError` error is constructed.

For example, if coin is QRC20 and there are no QTUM enough, `TradePreimageError::NotSufficientBalance::coin` has to be `QTUM`.